manifests/t/TominLab/WonderPen/2.1.10/TominLab.WonderPen.installer.yaml

```diff
@@ -41,4 +40,4 @@ Installers:
   ProductCode: afddeeee-0891-5662-a440-4865c9a9ea84
   ElevationRequirement: elevationRequired
```

Comment:
```suggestion
  ElevationRequirement: elevatesSelf
```

Reply:
GitHub's single comment suggestion doesn't seem to work after unfolding the page to reveal the other replacement location. So maybe @SpecterShell will have to modify that one.